.github/workflows/ci.yml

```diff
@@ -25,19 +25,22 @@ jobs:
       - uses: actions/setup-python@v5
         with:
           python-version: "3.11"
-      - name: Install pre-commit
-        run: python -m pip install pre-commit
-      - name: Show pre-commit requirements
-        run: python -m pip freeze --local
+      - name: Install just
+        run: ./ci/scripts/install-just.sh
       - uses: actions/cache@v4
         with:
           path: ~/.cache/pre-commit
           key: pre-commit-3|${{ env.pythonLocation }}|${{ hashFiles('.pre-commit-config.yaml') }}
-      - name: Run pre-commit for all files
-        run: pre-commit run --config './.pre-commit-config.yaml' --all-files --color=always --show-diff-on-failure
+      - name: Run pre-commit
+        run: just lint
+        shell: bash
 
   test-datastore:
     runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      issues: write
+      pull-requests: write
```

Comment:
What does this do?

Reply:
It is necessary for the `MishaKav/pytest-coverage-comment@main` which used to be a separate job. Using these permissions here is not what we want, I'll look into separating them.